coderd/organizations.go

```diff
@@ -175,6 +177,9 @@ func (api *API) patchOrganization(rw http.ResponseWriter, r *http.Request) {
 		if req.Description != "" {
 			updateOrgParams.Description = req.Description
 		}
+		if req.Icon != "" {
+			updateOrgParams.Icon = req.Icon
+		}
```

Comment:
One thing I noticed is this does not allow unsetting the icon/name/description. It's something we may want to add back in later. If in the UI you leave it blank, it will just default to the previous value.

We can use `*string` to distinguish "no change" from "empty input". Given the other fields are like this too, I won't block an approval on it.

Reply:
oh, right 💀 yeah, `Description` should also allow emptying

Reply:
We can follow up with this. A lot of API ux will likely be updated as we start dogfooding it.